csi/deploy/kubernetes/file/csi-nodeplugin-opensdsplugin.yaml

```diff
@@ -171,4 +171,4 @@ spec:
         - name: hosts
           hostPath:
              path: /etc
-             type: Directory
\ No newline at end of file
+             type: Directory
```

Comment:
'Directory' or 'DirectoryOrCreate' ??

Reply:
/etc should be created by default right?

Reply:
Fine..